ocs_ci/ocs/resources/objectconfigfile.py

```diff
@@ -0,0 +1,122 @@
+# -*- coding: utf8 -*-
+"""
+Representation of general Kubernetes/OpenShift objects config file.
+
+This allows one to work with multiple objects of different kind at once, as
+explained in `Imperative Management of Kubernetes Objects Using Configuration
+Files
+<https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-config/>`_.
+
+Usage:
+    First you prepare list of dictionaries of k8s objects such as Deployment or
+    PVC which describes your workload/project to be deployed in OCP. Then
+    create instance of ``ObjectConfFile`` class with the list to be able to
+    create the resource in the cluster (to run it), or delete it later when
+    it's no longer needed.
+"""
+
+
+import logging
+import os
+
+import yaml
+
+from ocs_ci.utility.utils import run_cmd
+
+
+LOGGER = logging.getLogger(name=__file__)
+
+
+class ObjectConfFile():
+    """
+    This class represents particular k8s object config file which describes
+    multiple k8s resources.
+
+    Methods of this class implements `Imperative Management of Kubernetes
+    Objects Using Configuration Files
+    <https://kubernetes.io/docs/tasks/manage-kubernetes-objects/imperative-config/>`_.
+    """
+
+    def __init__(self, name, obj_dict_list, project, tmp_path):
+        """
+        Args:
+            name (str): name of this object config file
```

Comment:
Capital 'N' at the beginning of the description.

Please fix all

Reply:
ok